### PR TITLE
fix file path issue

### PR DESCRIPTION
**Pull request by bendnuts**

### CODE REVIEW

File: pyndustric/__main__.py [hunk 1]
```diff
@@ -1,52 +1,57 @@
...
-        masm = pyndustric.Compiler().compile(source)
-        complete_masm += masm
-        took = time.time() - start
-        print(masm)
-        print(
-            f"# compiled {file} with pyndustric {pyndustric.__version__} in {took:.2f}s",
-            file=sys.stderr,
-        )
+    # for file in args.files:
+    file = args.file
```
Comment (Lonami) [inline pyndustric/__main__.py:29]: The old code was supposed to support compiling multiple files at the same time, and I would like to retain the old (intended) behaviour.

File: pyndustric/__main__.py [hunk 2]
```diff
@@ -1,52 +1,57 @@
...
-        start = time.time()
-        masm = pyndustric.Compiler().compile(source)
-        complete_masm += masm
-        took = time.time() - start
-        print(masm)
-        print(
-            f"# compiled {file} with pyndustric {pyndustric.__version__} in {took:.2f}s",
-            file=sys.stderr,
-        )
+    # for file in args.files:
```
Comment (Lonami) [inline pyndustric/__main__.py:28]: There is no need to keep the old code around in a comment. `git` fulfills this need.

File: pyndustric/__main__.py [hunk 3]
```diff
@@ -1,52 +1,57 @@
...
+        f"# compiled {file} with pyndustric {pyndustric.__version__} in {took:.2f}s",
+        file=sys.stderr,
+    )
 
     if args.clipboard:
-        import ait
+        try:
+            import ait
+            ait.copy(complete_masm)
+        except:
```
Comment (Lonami) [inline pyndustric/__main__.py:52]: Bare excepts are a bad idea. If `copy` failed we would think `autoit` is missing when it's not. We should at least use `except ImportError`.

File: pyndustric/__main__.py [hunk 4]
```diff
@@ -1,52 +1,57 @@
...
+            print("Require python module: autoit")
```
Comment (Lonami) [inline pyndustric/__main__.py:53]: While this is a nice addition, it's not related to fixing the issue with files, so I would prefer if it was a separate PR.

**Review by Lonami — CHANGES_REQUESTED** [review]
Thanks for the PR, unfortunately I think it needs some changes.

File: pyndustric/__main__.py (hunk 1 above)
Comment (bendnuts) [reply pyndustric/__main__.py]: i re-added the old behaviour

**Review by Lonami — APPROVED** [review]

**Comment by Lonami** [comment]
Thanks.

**Comment by bendnuts** [comment]
no problem

**Pull request merged**